website/views/shared/tasks/task.jade

```diff
@@ -133,10 +139,10 @@ li(bindonce='list', bo-id='"task-"+task.id', ng-repeat='task in obj[list.type+"s
               span=env.t('addChecklist')
         form.checklist-form(ng-if='task.checklist')
           fieldset.option-group(ng-if='!$state.includes("options.social.challenges")')
-            legend.option-title
+            legend.option-title(ng-if='task.checklist[0]')
```

Comment:
Yes, it hides the title containing the explanation until a checklist is added. I changed it for compactness. We could replace it with an inline "?" hint and save space. Otherwise I can just revert this line.